modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationEndpoints/EditAPIApplicationEndpoint.testcase

```diff
@@ -35,6 +35,67 @@ definition {
 		}
 	}
 
+	@priority = 3
+	test CanChangeSchemaOfEndpoint {
+		property portal.acceptance = "true";
+
+		task ("Given published API Application with Endpoint created") {
```

Comment:
We already create an api application with endpoint in the Setup() so there is no need to create it again, unless we need some other configs api application and endpoint